contracts/ModelRepository.sol

```diff
@@ -73,14 +73,22 @@ contract ModelRepository {
     models.push(newModel);
   }
 
+  function incentiveCalculate(uint bounty, uint totalError, uint solvedError) returns(uint total) {
```

Comment:
can we mark this function `constant` see `getNumModels` then e.g. wallet software could call the function without a transaction if its just interested in learning "what would my incentive be?"

Reply:
Yes, I will mark this function constant. Coming to your example, the software can call the function without transaction and calculate the incentive but it should do simple calculation to those variables (bestError,TargetError, InitialError and bounty) which are available and pass it to the function parameters. 
+ incentiveCalculate(bounty, initialError - targeterror, bestError - newError)